app/build.gradle

```diff
@@ -68,30 +68,16 @@ android {
 dependencies {
 
     // Import the Compose BOM
-    implementation platform('androidx.compose:compose-bom:2023.03.00')
+    implementation platform('androidx.compose:compose-bom:2023.04.01')
```

Comment:
i believe .05 is out

Reply:
not according to this page
https://developer.android.com/jetpack/compose/bom